core/src/main/java/bisq/core/dao/state/DaoStateService.java

```diff
@@ -811,6 +816,27 @@ public long getTotalAmountOfConfiscatedTxOutputs() {
                 .sum();
     }
 
+    public long getBurnedBsqOfAllInvalidTxs() {
+        return getTxStream()
+                .filter(e -> e.getTxType() == TxType.INVALID)
+                .mapToLong(this::getBurnedBsqOfInvalidTx)
+                .sum();
+    }
+
+    public long getBurnedBsqOfInvalidTx(Tx tx) {
```

Comment:
Hm... I consider it burned BSQ as well in contrast to burnedFee.

Reply:
Yes, my point was that the function can calculate the input value from any tx, not only invalid ones.

Reply:
Ah ok. I am not sure if it is valid for normal txs as well as all special cases handled in the input parser are ignored here. It only checks the txoutput types. I think for the use case of invalid txs that should be good enough. A maybe more safe approach would be to add another field to tx like burnedFee for the burned BSQ from an invalid tx. Only at parse time we have all checks done and we could not do that later as well. Hm... not sure if its worth to add that field but might be more safe. In fact we need to add a comment to that method that this is not intended to be used otherwise as it is not a perfect check....

Reply:
Yeah, good point. If we can add the field and set in the parser it would be more correct and safer I think.

Reply:
Implemented as discussed. We use the burntFee field in Tx but renamed it.